pkg/ccl/crosscluster/logical/dead_letter_queue_test.go

```diff
@@ -134,50 +134,56 @@ func TestDLQClient(t *testing.T) {
 
 	dbAName := "a"
 
-	tableNames := []fullyQualifiedTableName{
+	dstTableMd := []dstTableMetadata{
 		{
 			database: defaultDbName,
 			schema:   publicScName,
 			table:    "foo",
+			tableID:  1,
 		},
 		{
 			database: defaultDbName,
 			schema:   "baz",
 			table:    "foo",
+			tableID:  2,
 		},
 		{
 			database: defaultDbName,
 			schema:   "bar",
 			table:    "_foo",
+			tableID:  3,
 		},
 		{
 			database: defaultDbName,
 			schema:   "bar_",
 			table:    "foo",
+			tableID:  4,
 		},
 		{
 			database: dbAName,
 			schema:   publicScName,
 			table:    "bar",
+			tableID:  5,
 		},
 		{
 			database: dbAName,
 			schema:   "baz",
 			table:    "foo",
+			tableID:  6,
```

Comment:
because the tableID value of 6 doesn't add more coverage than the value of 5, you could just set to `const testTableID =1` for each case.

Reply:
I changed the IDs to 1 except for one test case where we need different IDs to create unique table names, i.e., `dlq_1_bar__foo` (schema name `bar_` and table name `foo`) and `dlq_2_bar__foo` (schema name `bar` and table name `_foo`).